src/official-account/official-account.ts

```diff
@@ -1,36 +1,40 @@
 /* eslint-disable camelcase */
 import { EventEmitter } from 'events'
+import { v4 } from 'uuid'
 
 import crypto from 'crypto'
-import { ContactGender, FileBox, log }  from 'wechaty-puppet'
+import { ContactGender, FileBox, log } from 'wechaty-puppet'
 import { FileBoxType } from 'file-box'
 
 import { normalizeFileBox } from './normalize-file-box'
 
 import {
   Webhook,
   VerifyArgs,
-}                      from './webhook'
+} from './webhook'
 import { PayloadStore } from './payload-store'
 import {
   getSimpleUnirest,
   SimpleUnirest,
-}                       from './simple-unirest'
+} from './simple-unirest'
 import {
   OAMessageType,
   // OAMediaPayload,
   OAMediaType,
   ErrorPayload,
   OAContactPayload,
-}                       from './schema'
+  OATagPayload,
+  OAMessagePayload,
+} from './schema'
```

Comment:
Could we keep the spaces for better readability? Because I feel that each import should be a separate block visually.